pkg/persistence/object_store.go

```diff
@@ -56,6 +58,7 @@ type BackupStore interface {
 	GetBackupVolumeSnapshots(name string) ([]*volume.Snapshot, error)
 	GetPodVolumeBackups(name string) ([]*velerov1api.PodVolumeBackup, error)
 	GetBackupContents(name string) (io.ReadCloser, error)
+	// TODO(nrb-csi): Do we need Get methods for the CSI types?
```

Comment:
I think we will for the CLI commands and the backup sync controller, but can add them in those PRs.